frontend/trpc/routes/users/index.ts

```
@@ -140,6 +221,42 @@ export const usersRouter = createRouter({
       contractSignedElsewhere: ctx.companyContractor.contractSignedElsewhere,
     };
   }),
+
+  getContractorStatus: companyProcedure
+    .input(z.object({ companyId: z.string() }))
+    .query(async ({ ctx, input }) => {
```

ESLint

[error] Replace `⏎····.input(z.object({·companyId:·z.string()·}))⏎····` with `.input(z.object({·companyId:·z.string()·}))`

(prettier/prettier)